org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/StaticInstanceReader.java

```diff
@@ -46,6 +46,8 @@
  * This utility class parses an XML file into an AlloyInstance object.
  * <p>
  * <b>Thread Safety:</b> Can be called only by the AWT event thread.
+ *
+ * @modified: Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
Creates an AlloyInstance for a particular state after parsing a the XML of a full Instance trace (expressions evaluated in that state); whether elements are mutable is also registered

Reply:
Consider moving some of these comments you left in the PR to the text of the `@modified` annotations